INSTALL.md

```diff
@@ -6,6 +6,9 @@ While not strictly required when connecting to a remote chain, we recommend that
 
 - [Foundry](https://book.getfoundry.sh/getting-started/installation) for running a local Anvil testnnet node.
 - [Docker](https://docs.docker.com/engine/install/) for hosting the underlying data tools.
+- [Rust](https://www.rust-lang.org/tools/install) for building hyperdrivepy
+  
+❗️ Ensure you are using rustc 1.78.0-nightly or newer
```

Comment:
We may want to put that this is only necessary if we don't have a prebuilt binary in pypi. The binaries we support are:
- osx arm
- osx x86_64
- linux w/ glibc >= 2.34